### PR TITLE
Refactor releases

### PR DESCRIPTION
**Pull request by selaux**
Goal is to:

- Create Bundle file for macOS
- Create Installer for Windows

To make it easier for users to install releases, which start the launcher. This should result in a easy setup experience for users, without the need to touch any config files manually.

I needed to create the PR early to make the CI build packages.

### CODE REVIEW

**Comment by selaux** [comment]
Please check if the manfile update wording is okay 🙂

File: assets/distr-files-linux/ja2_manpage [hunk 1]
```diff
@@ -10,6 +10,8 @@ JA2 Stracciatella is a cross-platform implementation of Jagged Alliance 2 game.
 This is only the game engine.  Game resources from the original JA2 are required to play the game.
 .TP
 When you first start the game, file \fB~/.ja2/ja2.json\fR is created.  You should edit this file, so that \fBdata_dir\fR points to the directory with game resources from the original JA2 game.
+.TP
+You can also use ja2-launcher to configure your game with a UI.
```
Comment (Manuel-K) [inline assets/distr-files-linux/ja2_manpage]: … use ja2-launcher to comfortably configure the game.

File: assets/distr-files-linux/ja2_manpage [hunk 2]
```diff
@@ -22,8 +24,9 @@ Screen resolution, e.g. 800x600. Default value is 640x480
 Version of the game resources.
 Possible values are: DUTCH, ENGLISH, FRENCH, GERMAN, ITALIAN, POLISH, RUSSIAN, RUSSIAN_GOLD.
 Default value is ENGLISH.
-RUSSIAN is for BUKA Agonia Vlasty release.
-RUSSIAN_GOLD is for Gold release.
+Use RUSSIAN for the BUKA Agonia Vlasty release.
```
Comment (Manuel-K) [inline assets/distr-files-linux/ja2_manpage]: Quotes around BUKA Agonia Vlasty

**Comment by Manuel-K** [comment]
@selaux This seems to be complete… right?

**Comment by selaux** [comment]
Yes. It could be made nicer, but I think it will do for now.

**Comment by Manuel-K** [comment]
Should I include this in the changelog as  `Maintenance: Refactor build system and update documentation (#629)`?

**Pull request merged**